### PR TITLE
Change log v46.16.0

### PR DESCRIPTION
**Pull request by sfsholden**
### What does this PR do?
Update to changelog

### CODE REVIEW

File: packages/salesforcedx-sobjects-faux-generator/src/constants.ts [hunk 1]
```diff
@@ -9,3 +9,5 @@ export const CLIENT_ID = 'sfdx-vscode';
 export const SFDX_DIR = '.sfdx';
 export const TOOLS_DIR = 'tools';
 export const SOBJECTS_DIR = 'sobjects';
+export const STANDARDOBJECTS_DIR = 'standardObjects';
```
Comment (lcampos) [inline packages/salesforcedx-sobjects-faux-generator/src/constants.ts]: These changes shouldn't be part of this PR.

Comment (sfsholden) [reply packages/salesforcedx-sobjects-faux-generator/src/constants.ts]: Thanks, I didn't double check the base branch

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
+# 46.16.0 - September 26, 2019
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Retrieve Multiple Components by Metadata Type ([PR #1667](https://github.com/forcedotcom/salesforcedx-vscode/pull/1667))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
- Add retrieve all instances of a single metadata type for the Org Browser ([PR #1667](https://github.com/forcedotcom/salesforcedx-vscode/pull/1667))
```

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md]: This is how we've called it in the Iteration Plan https://github.com/forcedotcom/salesforcedx-vscode/issues/1613

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+
+#### salesforcedx-sobjects-faux-generator, salesforcedx-vscode-apex
+
+- Select SObject category for Refresh SObject Definitions command ([PR #1681](https://github.com/forcedotcom/salesforcedx-vscode/pull/1681))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
- Select SObject category when running `SFDX: Refresh SObject Definitions` command ([PR #1681](https://github.com/forcedotcom/salesforcedx-vscode/pull/1681))
```

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 2 above)
Comment (hbc-bindu) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Could rephrase as:
Update Org Browser to retrieve all instances of a metadata type

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 3 above)
Comment (hbc-bindu) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Could rephrase as:
Allow selecting SObject category (custom, standard or all) when running SFDX: Refresh SObject Definitions

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+
+#### docs
+
+- Add retrieve multiple components documentation for [Org Browser](https://forcedotcom.github.io/salesforcedx-vscode/articles/user-guide/org-browser) ([PR #1667](https://github.com/forcedotcom/salesforcedx-vscode/pull/1667))
+
+- Add links at the bottom of the articles to provide feedback and log bugs ([PR #1646](https://github.com/forcedotcom/salesforcedx-vscode/pull/1646))
+
+- Add recommended timeout setting for [Prettier](https://forcedotcom.github.io/salesforcedx-vscode/articles/getting-started/prettier) setup ([PR #1647](https://github.com/forcedotcom/salesforcedx-vscode/pull/1647))
+
+- Added dropdown pre-selection setting info for use in [Apex](https://forcedotcom.github.io/salesforcedx-vscode/articles/apex/writing) ([PR #1665](https://github.com/forcedotcom/salesforcedx-vscode/pull/1665))
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Add dropdown list of pre-selection settings in [Apex](https://forcedotcom.github.io/salesforcedx-vscode/articles/apex/writing)

Comment (sfsholden) [reply packages/salesforcedx-vscode/CHANGELOG.md]: I don't think that sounds quite right. What about 'Add pre-selection settings for Apex auto-complete dropdown list'?

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,27 @@
...
+
+- Add retrieve multiple components documentation for [Org Browser](https://forcedotcom.github.io/salesforcedx-vscode/articles/user-guide/org-browser) ([PR #1667](https://github.com/forcedotcom/salesforcedx-vscode/pull/1667))
+
+- Add links at the bottom of the articles to provide feedback and log bugs ([PR #1646](https://github.com/forcedotcom/salesforcedx-vscode/pull/1646))
+
+- Add recommended timeout setting for [Prettier](https://forcedotcom.github.io/salesforcedx-vscode/articles/getting-started/prettier) setup ([PR #1647](https://github.com/forcedotcom/salesforcedx-vscode/pull/1647))
+
+- Add pre-selection settings for [Apex](https://forcedotcom.github.io/salesforcedx-vscode/articles/apex/writing) auto-complete dropdown list ([PR #1665](https://github.com/forcedotcom/salesforcedx-vscode/pull/1665))
+
+- Add [local develop](https://forcedotcom.github.io/salesforcedx-vscode/articles/lwc/localdev) documentation for lwc ([PR #1676](https://github.com/forcedotcom/salesforcedx-vscode/pull/1676)) —Contribution by [@blythesheldon](https://github.com/blythesheldon)
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
- Add [local develop](https://forcedotcom.github.io/salesforcedx-vscode/articles/lwc/localdev) documentation for LWC ([PR #1676](https://github.com/forcedotcom/salesforcedx-vscode/pull/1676)) —Contribution by [@blythesheldon](https://github.com/blythesheldon)
```
